samples/mediator.ts

```diff
@@ -13,6 +13,7 @@
  */
 
 import type { InitConfig } from '@aries-framework/core'
+import type { Socket } from 'net'
```

Comment:
It isn't the correct package to import `Socket`, is it?

Reply:
It is for the version of ws we have: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57590/files

This is fixed in v8 of ws, but there are some breaking changes that would need proper testing before updating (seems like there are some incompallities with the normal WS, but not sure)

So would like to keep it like this for now until we update to v8